### PR TITLE
feat: fallback image resolving for dev environments

### PR DESCRIPTION
**Pull request by GeneraleCauchemar**
- When in dev, checks if any path given leads to an existing image on the server, using Liip's DataManager logic. If not, a default image is rendered instead, to avoid NotLoadableException being triggered

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +81,14 @@ The following Twig functions are available in your templates.
 1. [umanit_image_figure](#umanit_image_figure)
 1. [umanit_image_picture_lazy_load](#umanit_image_picture_lazy_load)
 1. [umanit_image_picture](#umanit_image_picture)
+1. [umanit_image_img](#umanit_image_srcset)
```
Comment (DjLeChuck) [inline README.md]: Bad anchor

File: src/Runtime.php [hunk 2]
```diff
@@ -26,6 +27,8 @@ class Runtime
     private ImagineInterface $imagine;
     private bool $useLiipDefaultImage;
     private ?string $liipDefaultImage;
+    private string $env;
+    private FallbackImageResolver $FallbackImageResolver;
```
Comment (DjLeChuck) [inline src/Runtime.php:31]: No uppercase on first letter for a variable name

File: src/Runtime.php [hunk 3]
```diff
@@ -69,17 +76,18 @@ public function getImageFigure(
...
+        $nonLazyLoadImgMarkup = $this->getImageImg(
             $path,
             $srcFilter,
             $srcsetFilters,
             $alt,
             $imgClass,
             $sizes,
             $imgImportance,
-            $imgDataAttributes
+            $imgDataAttributes,
```
Comment (DjLeChuck) [inline src/Runtime.php]: I’m not sure finishing args list with a coma is working before PHP 8.0

**Review by DjLeChuck — CHANGES_REQUESTED** [review]

File: src/Runtime.php (hunk 3 above)
Comment (GeneraleCauchemar) [reply src/Runtime.php]: Fixed

File: src/Runtime.php (hunk 2 above)
Comment (GeneraleCauchemar) [reply src/Runtime.php:31]: Fixed (wrong search and replace)

File: README.md (hunk 1 above)
Comment (GeneraleCauchemar) [reply README.md]: Fixed

**Pull request closed without merging**